lib/sygus_converter.js

```diff
@@ -34,6 +34,35 @@ export function sygusToCode(sygusSolution, oldCode) {
   return newCode;
 };
 
+function getInputOutputType(intendedExamples, fxn){
+  var paramTypes = {};
+
+  var types = new Array();
+  var outputType;
+  var conversionDict = {"number": "Int", "string": "String", "boolean": "Bool"};
```

Comment:
```suggestion
  var typeofToSmtTypeMap = {"number": "Int", "string": "String", "boolean": "Bool"};
```

Reply:
gotta change it in other places too, but good to be clear about what the function of the variable is